license.md

```diff
@@ -1,23 +1,7 @@
-Copyright (c) 2015, SilverStripe
-All rights reserved.
+Copyright (c) 2015 Christopher Pitt
```

Comment:
The previous version of TagField used this license and owner. When I rewrote it (the rewrite which became version 2) I didn't assign the proper ownership or license. The same change happened with Blog v2 vs. v1...